src/libp2p_crypto.erl

```diff
@@ -370,10 +521,360 @@ base58check_decode(B58) ->
             {error, bad_checksum}
     end.
 
+-spec pubkey_is_multisig(pubkey()) -> boolean().
+pubkey_is_multisig({multisig, _, _, _}) ->
+    true;
+pubkey_is_multisig({ecc_compact, _}) ->
+    false;
+pubkey_is_multisig({ed25519, _}) ->
+    false.
+
+%% @doc The binary form of this multisig-pubkey can be optained with
+%% pubkey_to_bin (just as any other pubkey), the format of which will be
+%% roughly:
+%%
+%%     <<NetType:4, KeyType:4, M/integer, N/integer, KeysDigest/binary>>
```

Comment:
What's the relationship between M and N? if any...

Reply:
Nevermind, I see that `M` goes from `1 -> N` in test generation.

Reply:
@vihu do you mean on the higher, semantic level? If so, it is the M minimal sigs required out of N total keys. This info will be in a HIP.